### PR TITLE
added powershell hardening

### PR DESCRIPTION
**Pull request by MattTunny**

### CODE REVIEW

File: recipes/powershell.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#
+# Cookbook Name:: windows-hardening
+# Recipe:: powershell
+#
+# Copyright (c) 2016 Joe Gardiner, All Rights Reserved.
```
Comment (chris-rock) [inline recipes/powershell.rb]: I assume this is copy&paste error

File: recipes/powershell.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+# Enabling PowerShell script block logging will record detailed information from the processing of PowerShell commands and scripts
+# Powershell ScriptBlock Logging
+registry_key 'HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Microsoft\Windows\PowerShell\ScriptBlockLogging' do
```
Comment (chris-rock) [inline recipes/powershell.rb]: Can you reference where those requirements are coming from? We need to update our baseline then too https://github.com/dev-sec/windows-baseline

File: recipes/powershell.rb (hunk 1 above)
Comment (MattTunny) [inline recipes/powershell.rb]: yeah, i'll fix up

File: recipes/powershell.rb (hunk 2 above)
Comment (MattTunny) [inline recipes/powershell.rb]: they are fairly new but very helpful, sort of same thing as bash history for linux it enables administrators to see any PowerShell commands that have been run on the box and log into event logs. You can optionally move it into a different log file but I think we should leave it in the default event logs as most people off load those anyway. 
Theres a few whitepapers on it from microsoft, heres the powershell team explain it: https://blogs.msdn.microsoft.com/powershell/2015/06/09/powershell-the-blue-team/

I'm submitting the pull request for windows-baseline today

**Comment by MattTunny** [comment]
added powershell logging to windows-baseline https://github.com/dev-sec/windows-baseline/pull/9

**Comment by MattTunny** [comment]
Sorry forgot to add CIS link for these 2 settings:
https://benchmarks.cisecurity.org/tools2/windows/CIS_Microsoft_Windows_Server_2012_R2_Benchmark_v2.2.1.pdf

18.9.84 Windows PowerShell
18.9.84.1 (L1) Ensure 'Turn on PowerShell Script Block Logging' is set to
'Disabled' (Scored) - Page 654
18.9.84.2 (L1) Ensure 'Turn on PowerShell Transcription' is set to
'Disabled' (Scored) - Page 656

**Comment by chris-rock** [comment]
@MattTunny Can you sign-off this commit? git commit --amend -s would do the trick. Once this is done, LGTM

**Comment by MattTunny** [comment]
ok done

**Comment by chris-rock** [comment]
Thank you @MattTunny for this great improvement.

**Pull request merged**